scripts/repl.js

```diff
@@ -57,6 +62,86 @@
   };
 
   /*
+   * Utils for working with parse.com for persistence
+   */
+  function ParseUtils () {}
+
+  var __parseLoadingCallbacks = null;
```

Comment:
It's always constructed here:

```js
function REPL () {
  this.shortener = new ParseUtils();
```

And conditionally here:
```js
if (initialQuery && initialQuery.short) {
  new ParseUtils().get(initialQuery.short, function (err, data) {
```

It could be done in an iife to create a singleton if that's preferred.  I was just trying to stick to the existing style as much as possible.